graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/CodeGenTest.kt

```diff
@@ -2598,6 +2598,117 @@ class CodeGenTest {
         assertCompilesJava(dataTypes + interfaces)
     }
 
+    @Test
+    fun generateDataClassWithBitset() {
```

Comment:
should we also add a test for when a data class is generated without a bitset?

Reply:
Thanks @eaugeas, added an additional test without a bitset